--- a/src/dialogs.py    Wed Aug 24 15:59:06 2011 +0200
+++ b/src/dialogs.py    Wed Apr 20 22:57:26 2011 +0200
@@ -994,8 +994,9 @@

         if self.account:
             message_buffer = self.message_textview.get_buffer()
-            message_buffer.set_text(helpers.get_subscription_request_msg(
+            msg = helpers.from_one_line(helpers.get_subscription_request_msg(
                 self.account))
+            message_buffer.set_text(msg)

     def on_add_new_contact_window_destroy(self, widget):
         if self.account:
@@ -1074,8 +1075,9 @@
             end_iter = message_buffer.get_end_iter()
             message = message_buffer.get_text(start_iter, end_iter).decode('utf-8')
             if self.save_message_checkbutton.get_active():
+                msg = helpers.to_one_line(message)
                 gajim.config.set_per('accounts', self.account,
-                        'subscription_request_msg', message)
+                        'subscription_request_msg', msg)
         else:
             message= ''
         group = self.group_comboboxentry.child.get_text().decode('utf-8')
