Index: gajim-0.14/src/common/passwords.py
===================================================================
--- gajim-0.14.orig/src/common/passwords.py	2010-09-06 14:00:45.329749782 +0200
+++ gajim-0.14/src/common/passwords.py	2010-09-06 14:00:52.429755870 +0200
@@ -63,7 +63,7 @@
     def __init__(self):
         self.keyring = gnomekeyring.get_default_keyring_sync()
         if self.keyring is None:
-            self.keyring = 'default'
+            self.keyring = 'login'
         try:
             gnomekeyring.create_sync(self.keyring, None)
         except gnomekeyring.AlreadyExistsError:
